articles/tokens/refresh-token.md

```diff
@@ -1,22 +1,25 @@
 ---
-description: A refresh token allows an application to request Auth0 to issue a new id_token directly, without needing to re-authenticate the user.
+title: Refresh Token
+description: What is a Refresh Token and what you can do with it.
 ---
 
-# Refresh Tokens
+# Refresh Token
 
-A **refresh token** is a special kind of [JWT](/jwt) that is used to authenticate a user without them needing to re-authenticate. This is primarily useful for mobile applications that are installed on a device.  
+A **refresh token** is a special kind of [JWT](/jwt) that contains the information required to obtain a new `access_token` or `id_token`.
 
-For more information on the types of access tokens used by Auth0, see [Tokens](/tokens).
+> An `id_token` is used by the client that receives it in order to identify the user. An `access_token` is used by a client to access an API. For more information refer to: [Tokens](/tokens).
 
-If you are new to refresh tokens, you can learn more about them in this blog post: [Refresh Tokens: When to Use Them and How They Interact with JWTs](https://auth0.com/blog/refresh-tokens-what-are-they-and-when-to-use-them/).
+Usually, a user will need a new access token only after the previous one expires, or when gaining access to a new resource for the first time.
 
-## Introduction
+Refresh tokens are subject to strict storage requirements to ensure that they are not leaked. Refresh tokens can also be blacklisted by the authorization server.
```

Comment:
This should link to the doc that describes how to revoke refresh tokens.

We should be consistent in our usage of "revoking" vs "blacklisting". The OIDC spec uses "revoke", so I would suggest we use that (see example use here: https://openid.net/specs/openid-connect-core-1_0.html#TokenLifetime)